File: Service/ClientCare/RMA.Service.ClientCare.Contracts/Entities/Quote/QuoteAllowance.cs
using RMA.Service.Admin.MasterDataManager.Contracts.Enums;

namespace RMA.Service.ClientCare.Contracts.Entities.Quote
{
    public class QuoteAllowance
    {
        public int QuoteAllowanceId { get; set; }
        public int QuoteId { get; set; }
        public AllowanceTypeEnum AllowanceType { get; set; }
        public decimal? Allowance { get; set; }
    }
}

File: Web/RMA-Web-Shell/projects/clientcare/src/app/quote-manager/models/quote-allowance.ts
import { AllowanceTypeEnum } from 'projects/shared-models-lib/src/lib/enums/allowance-type-enum';

export class QuoteAllowance {
  quoteAllowanceId: number;
  quoteId: number;
  allowanceType: AllowanceTypeEnum;
  allowance: number;
}